### ISSUE TITLE
Unable to install sitespeed.io on node 10 since 12.2.0 (switch to coach-core)

### ISSUE THREAD

**Issue by drptbl** [sitespeedio/sitespeed.io#2939]
Hey @soulgalore. After #2912 change I'm not able to use sitespeed.io on node 10, also I can't find any information about dropping support for node 10 in sitespeed.io changelog. Is there any chance to fix it? Thanks :pray:.

```
 error coach-core@5.0.0: The engine "node" is incompatible with this module. Expected version ">=12.16.1". Got "10.19.0"
 error Found incompatible module.
```

**Comment by soulgalore** [comment]
Hi @drptbl sorry my mistake, I've fixed it now in 12.3.0. I'll remove 10 support when it reaches EOL next year or move all over all tools at the same time.

**Comment by drptbl** [comment]
It's fixed now. Thanks @soulgalore 🎉.
